card/mod/05_standard/spec/set/self/signin_spec.rb

```diff
@@ -1,73 +1,69 @@
 # -*- encoding : utf-8 -*-
 
-#FIXME need more specific assertions
+# FIXME need more specific assertions
```

Comment:
Annotation keywords should be all upper case, followed by a colon and a space, then a note describing the problem.